test/format/format_config.c

```diff
@@ -1449,6 +1451,90 @@ config_statistics(void)
     }
 }
 
+/*
+ * config_chunk_cache --
+ *     Chunk cache configuration.
+ */
+static void
+config_chunk_cache(void)
+{
+    char buf[128];
+    const char *chunkcache_type;
+    const char *storage_source;
+
+    storage_source = GVS(TIERED_STORAGE_STORAGE_SOURCE);
+    chunkcache_type = NULL;
+
+    /* Chunkcache does not work unless tiered storage is configured with dir_store. */
+    if (!g.tiered_storage_config || strcmp(storage_source, "dir_store") != 0) {
```

Comment:
At some point, we absolutely want to run the chunk cache with s3_store and all the others.  Is there a reason this is restricted now?  I would think we'd probably want to test it with s3 before long.

Reply:
Forgot to remove this, thanks for pointing out, updated in [b23ede9](https://github.com/wiredtiger/wiredtiger/pull/9589/commits/b23ede907b04cbd04fe0ee37772262b718a26e35)